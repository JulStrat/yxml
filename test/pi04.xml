<?xml version="1.1"?><?XM?><a/>
