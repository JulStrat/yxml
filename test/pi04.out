
pistart XM
piend
elemstart a
content
elemend
ok
